server/container/guice/distributed/src/main/java/org/apache/james/modules/blobstore/BlobStoreConfiguration.java

```diff
@@ -108,7 +112,8 @@ public static RequireImplementation builder() {
     public enum BlobStoreImplName {
         CASSANDRA("cassandra"),
         FILE("file"),
-        S3("s3");
+        S3("s3"),
```

Comment:
Can we remove the "cassandra" option please?

Reply:
class `BlobStoreConfiguration` is common, it is not used in postgres-app

Reply:
Might want a refactoring then, including in the master branch

Reply:
No then forget about that it is OK.

I misread and though this was a PG specific location. Sorry...